eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/threading/PollingThread.java

```diff
@@ -0,0 +1,353 @@
+package com.hms_networks.americas.sc.ignition.threading;
+
+import com.hms_networks.americas.sc.ignition.config.EwonConnectorSettings;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A thread that polls at a specified interval. The polling logic is defined by the implementation
+ * of the {@link PollingThread#run()} method. The polling interval can be changed at runtime by
+ * calling the {@link PollingThread#setPollingInterval(long, TimeUnit)} method.
+ *
+ * @author HMS Networks, MU Americas Solution Center
+ * @since 2.0.0
+ * @version 1.0.0
```

Comment:
same thing here

Reply:
See comment above: https://github.com/hms-networks/IgnitionEwonConnector/pull/141#discussion_r1218028830

The initial commit set these values incorrectly.